providers/implementations/include/prov/names.h

```diff
@@ -384,11 +384,11 @@
 #define PROV_DESCS_RSA_PSS "OpenSSL RSA-PSS implementation"
 #define PROV_NAMES_SM2 "SM2:1.2.156.10197.1.301"
 #define PROV_DESCS_SM2 "OpenSSL SM2 implementation"
-#define PROV_NAMES_ML_KEM_512 "ML-KEM-512"
+#define PROV_NAMES_ML_KEM_512 "ML-KEM-512:MLKEM512"
```

Comment:
FWIW, I'll also be adding the OID as a string to these in the encoders PR, so whoever is merged last will have to deal with a conflict. But this is not a blocker to approval.